Tutorials/WinForms/Classic/bind-the-data-grid-to-large-data-source.md

```diff
@@ -0,0 +1,301 @@
+                }
+            }
+
+            private void btnReload_Click(object sender, EventArgs e) {
+                Reload();
+            }
+        }
+    }
+    ```
+    </details>
+## Create the orders list Form
+* Select the **CustomersListForm** item in the Solution Explorer window and press **Ctrl+C** to copy the Form.
+* Press **Ctrl+V** to create a copy of a CustomersListForm copy and rename it to **OrdersListForm**.
+* Change the class and constructor name to **OrdersListForm**.
+* Open the `OrdersListForm` designer and change the `Text` property to **Orders**.
+* Select the **Events** tab page in the **Properties** window, right-click the `OrdersListForm.Load` event to invoke the context menu, and select the **Reset** menu item.
+* Add the [XPInstantFeedbackView](https://docs.devexpress.com/XPO/DevExpress.Xpo.XPInstantFeedbackView) component from the toolbox. Change its name to **OrdersInstantFeedbackView**. 
+  > The [XPInstantFeedbackView](https://docs.devexpress.com/XPO/DevExpress.Xpo.XPInstantFeedbackView) component improves performance when working with large tables. Refer to the [Large Data Sources: Server and Instant Feedback Modes](https://docs.devexpress.com/WindowsForms/8398/controls-and-libraries/data-grid/data-binding/large-data-sources-server-and-instant-feedback-modes) article for additional details.
+* Rebuild the project and set the `OrdersInstantFeedbackView.ObjectType` property to **DxSample.DataAccess.Order**.
+* Select the **Properties** property and click the ellipsis button to add the following properties:
+    * Name = Oid, Property = [Oid]
+    * Name = Product Name, Property = [ProductName]
+    * Name = Order Date, Property = [OrderDate]
+    * Name = Freight, Property = [Freight]
+* Delete the `CustomersBindingSource` component.
+* Change the `CustomersGridControl` name to **OrdersGridControl** and do the same for `CustomersGridView`.
+* Set the `OrdersGridControl.DataSource` property to **OrdersInstantFeedbackView**.
+* Select the `OrdersGridControl` control and click the [smart-tag](https://docs.microsoft.com/en-us/dotnet/framework/winforms/controls/performing-common-tasks-using-smart-tags-on-wf-controls) glyph. 
+* Select the **Run Designer** menu item.
+* Select the **Columns** item in the navigation panel and click the **Retrieve Fields** button in the toolbar.
+* Select the **Layout** item in the navigation panel, drag the **Oid** grid column down, and release it when the cross icon appears. 
+  > Do not delete this column from the [Columns](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.Views.Base.ColumnView.Columns) collection. It is required to retrieve record indentifiers using the [GetRowCellValue](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.Views.Grid.GridView.GetRowCellValue.overloads) method. Click **Apply**, close the designer, and save the changes.
```

Comment:
indentifiers?

Reply:
Do not use gerunds like 'using'

Reply:
>indentifiers?

The Oid column holds values which identify persistent objects. I can change "record identifiers" to "object identifiers" or change the sentence to "It holds identity values which required to show the Edit Form or delete an object".

Reply:
I also noticed my mistake: "indentifiers" => "identifiers"

Reply:
>Do not use gerunds like 'using'

Finally, changed this sentence to: "It holds object keys required to show the Edit Form or delete an object."